core/web/components/forms/app/add.tsx

```diff
@@ -1,6 +1,6 @@
 import { useState, useEffect } from "react";
 import { useApi } from "../../../hooks/useApi";
-import { Form, Button, Row, Col, Card } from "react-bootstrap";
+import { Form, Button, Row, Col, Card, CardColumns } from "react-bootstrap";
```

Comment:
I had no idea about `CardColumns`! Nice find!

Reply:
I think this means you shouldn't use `Row` and `Col` within the `CardColumns` section... it handles that all for you.

Reply:
For centering, it might be easiest to use tex-align rather then messing with the Float properties.  Perhaps on the body: `<Card.Body style={{textAlign: 'center'}}>`